Backpack-SwiftUI/Font/Classes/BPKFont.swift

```diff
@@ -0,0 +1,87 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+import SwiftUI
+
+public struct BPKFont {
+    static var fontDefinition: BPKFontDefinition?
+    
+    public static func setFontDefinition(_ fontDefinition: BPKFontDefinition) {
+        BPKFont.fontDefinition = fontDefinition
+    }
+}
+
+public protocol BPKFontDefinition {
+    var fontFamily: String { get }
+    var regularFontFace: String { get }
+    var semiboldFontFace: String { get }
+    var heavyFontFace: String { get }
```

Comment:
do we still need heavy?

Reply:
theres no heavy font left, guess its safe to remove!